modules/nw-metallb-installing-operator-cli.adoc

```diff
@@ -101,7 +99,7 @@ metadata:
   name: metallb-operator-sub
   namespace: metallb-system
 spec:
-  channel: "${OC_VERSION}"
+  channel: "stable"
```

Comment:
why this change?

Reply:
I checked and this command oc get packagemanifest metallb-operator -n openshift-marketplace -o jsonpath='{.status.defaultChannel} returned stable. I know around 4.10 timeframe some Operators were moved to stable channel. Maybe @evgenLevin can verify, but I recall making updates to other Operators from OC_VERSION to stable in 4.10 release

Reply:
I reverted this change so that it will substitute based on ```
OC_VERSION=$(oc version -o yaml | grep openshiftVersion | \
    grep -o '[0-9]*[.][0-9]*' | head -1)
```